<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoEditor</class>
 <widget class="QMainWindow" name="VideoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>619</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VideoEditor</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip { color: #000000; background-color: #232323; border: 0px; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #232323;
QToolTip { color: #ffffff; background-color: #232323; border: 0px; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="imgEditor" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #232323</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="imgThumbnail" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>12</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(35,35,35)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="audioListWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgb(51,51,51)</string>
            </property>
            <layout class="QGridLayout" name="gridLayout"/>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="imgListWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>10</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgb(51,51,51)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="imgPreview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>15</horstretch>
           <verstretch>20</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(51,51,51);
QToolTip { color: #000000; background-color: #ffffff; border: 0px; }</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="imgToolbar" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>10</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: #232323;
QToolTip { color: #000000; background-color: #ffffff; border: 0px; }</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,0,0,0">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>12</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="blurButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string comment="blur the image">Blur</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background: #eeeeee;
QToolTip { color: #000000; background-color: #eeeeee; border: 0px; }</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/img-blur.png</normaloff>:/img-blur.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="rotateButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Rotate</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background: #eeeeee;
QToolTip { color: #000000; background-color: #eeeeee; border: 0px; }</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/img-rotate.png</normaloff>:/img-rotate.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="brightButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Brightness&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background: #eeeeee;
QToolTip { color: #000000; background-color: #eeeeee; border: 0px; }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme=":/img-brightness.png">
                 <normaloff>:/img-brightness.png</normaloff>:/img-brightness.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>30</height>
                </size>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="resetButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Reset to Original</string>
               </property>
               <property name="styleSheet">
                <string notr="true">background: #eeeeee;
QToolTip { color: #000000; background-color: #eeeeee; border: 0px; }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme=":/img-reset.png">
                 <normaloff>:/img-reset.png</normaloff>:/img-reset.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>30</height>
                </size>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="padding" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="VideoPlayer" name="preview" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgb(35,35,35)</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="VideoWindow" name="videoWindow">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>7</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background: #dddddd</string>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="controller" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background: rgb(51,51,51)</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QToolButton" name="skipBackward">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="sizeIncrement">
                   <size>
                    <width>1</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: #eeeeee;</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="playButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: #eeeeee;</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="skipForward">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: #eeeeee;</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="ProgressBar" name="controlSlider">
      <property name="styleSheet">
       <string notr="true">background: #eeeeee</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Timeline" name="timeline">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(51,51,51); color: #ffffff;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Media"/>
    <addaction name="actionImport_Image"/>
    <addaction name="actionImport_Audio"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Preview</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPlay_from_Beginning"/>
    <addaction name="separator"/>
    <addaction name="actionBackward"/>
    <addaction name="actionForward"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionVEST_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
    <addaction name="actionV_CEST_Unique_Feature"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionBlur_Image"/>
    <addaction name="actionRotate_Image"/>
    <addaction name="actionBrighten_Image"/>
    <addaction name="actionReset_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAudio"/>
   <addaction name="menuImages"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play/Pause</string>
   </property>
  </action>
  <action name="actionPlay_from_Beginning">
   <property name="text">
    <string>Play from Beginning</string>
   </property>
  </action>
  <action name="actionVEST_Help">
   <property name="text">
    <string>VEST Help</string>
   </property>
  </action>
  <action name="actionImport_Image">
   <property name="text">
    <string>Import Image</string>
   </property>
  </action>
  <action name="actionImport_Audio">
   <property name="text">
    <string>Import Audio</string>
   </property>
  </action>
  <action name="actionImport_Media">
   <property name="text">
    <string>Import Media...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionBlur_Image">
   <property name="text">
    <string>Blur Image</string>
   </property>
  </action>
  <action name="actionBrighten_Image">
   <property name="text">
    <string>Brighten Image</string>
   </property>
  </action>
  <action name="actionRotate_Image">
   <property name="text">
    <string>Rotate Image</string>
   </property>
  </action>
  <action name="actionReset_Image">
   <property name="text">
    <string>Reset Image</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="text">
    <string>Backward</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionV_CEST_Unique_Feature">
   <property name="text">
    <string>V-CEST Unique Feature</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoPlayer</class>
   <extends>QWidget</extends>
   <header>videoplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QSlider</extends>
   <header>progressbar.h</header>
  </customwidget>
  <customwidget>
   <class>Timeline</class>
   <extends>QGraphicsView</extends>
   <header>timeline.h</header>
  </customwidget>
  <customwidget>
   <class>VideoWindow</class>
   <extends>QGraphicsView</extends>
   <header>videowindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
